qdrant-landing/content/documentation/frameworks/langchain.md

```diff
@@ -8,22 +8,22 @@ aliases:
 
 # LangChain
 
-LangChain is a library that makes developing Large Language Models based applications much easier. It unifies the interfaces 
-to different libraries, including major embedding providers and Qdrant. Using LangChain, you can focus on the business value 
+LangChain is a library that makes developing Large Language Models based applications much easier. It unifies the interfaces
+to different libraries, including major embedding providers and Qdrant. Using LangChain, you can focus on the business value
 instead of writing the boilerplate.
 
-Langchain comes with the Qdrant integration by default. It might be installed with pip:
+Langchain distributes their Qdrant integration in their community package. It might be installed with pip:
 
 ```bash
-pip install langchain
+pip install langchain-community
 ```
 
-Qdrant acts as a vector index that may store the embeddings with the documents used to generate them. There are various ways 
+Qdrant acts as a vector index that may store the embeddings with the documents used to generate them. There are various ways
 how to use it, but calling `Qdrant.from_texts` is probably the most straightforward way how to get started:
 
 ```python
-from langchain.vectorstores import Qdrant
-from langchain.embeddings import HuggingFaceEmbeddings
+from langchain_community.vectorstores import Qdrant
```

Comment:
How about replacing this import with `langchain-qdrant` instead? The community one is deprecated now.